tests/foreman/ui/test_ansible.py

```diff
@@ -0,0 +1,356 @@
+"""Tests for Ansible integration in UI
+
+@Requirement: Ansible Integration
+
+@CaseLevel: Acceptance
+
+@CaseComponent: UI
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+
+from robottelo.decorators import run_only_on, stubbed, tier1, tier3, tier4
+from robottelo.test import UITestCase
+
+
+class AnsibleTestCase(UITestCase):
+    """Implements Ansible integration tests in UI"""
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier4
+    def test_positive_install_ansible_plugin(self):
+        """Install Ansible plugin in Satellite
+
+        @id: 88b5bad8-7d15-4286-a70e-0c7125a0de39
+
+        @steps:
+
+        1. Install Ansible plugin in Satellite using
+           --enable-foreman-plugin-ansible option
+
+        @assert: Ansible plugin is installed succesfully. This can be verified
+        by editing any host and checking the presence of `Ansible Roles` tab.
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_setup_ansible_callback_1(self):
+        """Setup Ansible callback when Ansible is installed in the Satellite
+        host
+
+        @id: d9296383-66f8-4999-9102-ef28aff77877
+
+        @assert: Ansible callback is setup succesfully. This can be verified by
+        running Ansible setup module in any of the hosts.
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_setup_ansible_callback_2(self):
+        """Setup Ansible callback when Ansible is installed in the Capsule
+
+        @id: 72dec524-268e-480a-b7c1-6df2ebbe0d43
+
+        @assert: Ansible callback is setup succesfully. This can be verified by
+        running Ansible setup module in any of the hosts.
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_ansible_tower_1(self):
+        """Generate Ansible run reports when Ansible Tower is installed in the
+        Satellite host
+
+        @id: 45d1f7e2-1071-432c-a6cc-7757a26ca2e4
+
+        @steps:
+
+        1. Install Ansible Tower in the Satellite host
+        2. Setup Ansible callback in the Tower
+        3. Run Ansible setup module
+
+        @assert: Ansible Tower is integrated with Satellite and the Ansible run
```

Comment:
Such integration exists? **tower X satellite**? I understood that we can run ansible tower together with satellite in the same server but did not see any kind of integration. Did you mean that Satellite will consume some reports generated by tower?